cypress/e2e/yves/claim/claim.cy.ts

```diff
@@ -0,0 +1,313 @@
+    });
+
+    it('customer should be able to create and view a general claim', (): void => {
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+      claimListPage.clickCreateClaimButton();
+
+      staticFixtures.generalClaim.availableTypes = staticFixtures.claimTypes.general;
+      claimCreatePage.assertPageLocation();
+      claimCreatePage.createClaim(staticFixtures.generalClaim);
+
+      claimDetailPage.assertPageLocation();
+      cy.contains(claimCreatePage.getClaimCreatedMessage()).should('exist');
+
+      var reference: string;
+      cy.url()
+        .then((url) => {
+          const urlParams = new URLSearchParams(url.split('?')[1]);
+          reference = urlParams.get('reference');
+        })
+        .then(() => {
+          claimDetailPage.assertClaimDetails({
+            reference: reference,
+            type: staticFixtures.generalClaim.type,
+            subject: staticFixtures.generalClaim.subject,
+            description: staticFixtures.generalClaim.description,
+            status: staticFixtures.generalClaim.status,
+            files: staticFixtures.generalClaim.files,
+            date: new Date()
+              .toLocaleDateString('en-US', {
+                year: 'numeric',
+                month: 'short',
+                day: '2-digit',
+              })
+              .replace(/([a-zA-Z]+)\s/, '$1. '),
+            customer: {
+              firstName: dynamicFixtures.customer.first_name,
+              lastName: dynamicFixtures.customer.last_name,
+              email: dynamicFixtures.customer.email,
+              companyName: dynamicFixtures.company.name,
+              businessUnitName: dynamicFixtures.businessUnitFromCompany.name,
+            },
+          });
+        });
+    });
+
+    it('customer should be able to create and view an order claim', (): void => {
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      orderDetailPage.visit({
+        qs: {
+          id: dynamicFixtures.order.id_sales_order,
+        },
+      });
+      claimOrderPage.clickCreateClaimButton();
+
+      claimCreatePage.assertPageLocation();
+      staticFixtures.orderClaim.availableTypes = staticFixtures.claimTypes.order;
+      claimCreatePage.createOrderClaim({
+        ...staticFixtures.orderClaim,
+        orderReference: dynamicFixtures.order.order_reference,
+      });
+
+      claimDetailPage.assertPageLocation();
+      cy.contains(claimCreatePage.getClaimCreatedMessage()).should('exist');
+
+      var reference: string;
+      cy.url()
+        .then((url) => {
+          const urlParams = new URLSearchParams(url.split('?')[1]);
+          reference = urlParams.get('reference');
+        })
+        .then(() => {
```

Comment:
can't get what it is. Can we use smth like 

- claimListPage.openLatestClaimDetailsPage();
- laimDetailPage.assertOrderClaimDetails


?